app/database/models.py

```diff
@@ -29,6 +34,24 @@ class Event(Base):
     content = Column(String)
     start = Column(DateTime, nullable=False)
     end = Column(DateTime, nullable=False)
+    default_end = get_default_end_time()
```

Comment:
This class is part of the ORM, it must be of type `Column` or a `relationship`.

You can check the SQLAlchemy docs to see how to add default value to the `end` column.